src/views/SignBtcTransactionLedger.vue

```diff
@@ -92,6 +87,9 @@ export default class SignBtcTransactionLedger extends SignBtcTransaction {
     private _isDestroyed: boolean = false;
 
     protected async created() {
```

Comment:
No need for `async` here since we're not awaiting for bitcoinJs to be loaded.

Reply:
It's there to match the method signature of the parent class.